input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,265 @@ Example: Patient resource with interpreter required and language is known
+
+1\. Primary `code` only (pre-coordinated body site including laterality)
+* For systems that have pre-coordinated coding describing a concept fully.
+* Only the `code` element is used and contains information on body site with laterality.
+
+Example Condition resource cellulitis of right knee
+~~~
+{
+  "resourceType" : "Condition",
+  "id" : "cellulitis",
+  "clinicalStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code" : "active"
+      }
+    ]
+  },
+  "verificationStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code" : "confirmed"
+      }
+    ]
+  },
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code" : "encounter-diagnosis",
+          "display" : "Encounter Diagnosis"
+        }
+      ]
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://snomed.info/sct",
+        "code" : "10633311000119108",
+        "display" : "Cellulitis of right knee"
+      },
+      "text" : "Cellulitis of right knee"
+    ]
+  }
+}
+~~~
+
+2\. Primary `code` only (pre-coordinated body site without laterality and separate laterality qualifier)
+* For systems that have 
+  * Pre-coordinated coding describing a concept including body site without laterality
+  * Laterality qualifier recorded separately e.g. left, right
+* The `code` element is used
+  * Contains `coding` for primary concept (no body site information)
+  * Use `text` to describe concept fully, this can include information on recorded laterality e.g. ', Right'
+* Note: in this case laterality is not expressed in coded form 
+
+Example Condition resource cellulitis of knee, laterality as text only
+~~~
+{
+  "resourceType" : "Condition",
+  "id" : "cellulitis",
+  "clinicalStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code" : "active"
+      }
+    ]
+  },
+  "verificationStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code" : "confirmed"
+      }
+    ]
+  },
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code" : "encounter-diagnosis",
+          "display" : "Encounter Diagnosis"
+        }
+      ]
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://snomed.info/sct",
+        "code" : "13301002",
+        "display" : "Cellulitis of knee"
+      },
+      "text" : "Cellulitis of knee, Right"
+    ]
+  }
+}
+~~~
+
+3\. Primary `code` and `bodySite` with laterality coded separately.
+* For systems that have 
```

Comment:
Reformat this to be more similar to Missing Data advice so it becomes:

* For systems that have pre-coordinated coding describing primary concept without body site and separate body site with laterality recorded as coded value.
  * use the `code` element
    * `code.coding` contains the primary concept including body site without laterality.
    * `code.text` describes the concept fully, this can include information on recorded body site and laterality as text.
  * optionally, coded element `bodySite` may be supplied containing the coded body site with laterality